spdmlib/src/crypto/mod.rs

```diff
@@ -149,16 +149,16 @@ pub mod hmac {
     use super::CRYPTO_HMAC;
     use crate::crypto::SpdmHmac;
     use crate::error::{SpdmResult, SPDM_STATUS_VERIF_FAIL};
-    use crate::protocol::{SpdmBaseHashAlgo, SpdmDigestStruct};
+    use crate::protocol::{SpdmBaseHashAlgo, SpdmDigestStruct, SpdmFinishedKeyStruct};
 
     #[cfg(not(any(feature = "spdm-ring")))]
     static DEFAULT: SpdmHmac = SpdmHmac {
         hmac_cb: |_base_hash_algo: SpdmBaseHashAlgo,
-                  _key: &[u8],
+                  _key: &SpdmFinishedKeyStruct,
```

Comment:
This does not make sense to me.

Reply:
> This does not make sense to me.
> 
> HMAC function shall only follow HMAC specification. It should not know SpdmFinishedKey.

Yeah, agree . I just changed the SpdmFinishedKeyStruct to InputKeyMaterial in latest commit.